transactions/scripts/get_topshot_metadata.cdc

```diff
@@ -0,0 +1,147 @@
+    pub let draftSelection: String?
+    pub let draftRound: String?
+    pub let teamAtMomentNBAID: String?
+    pub let teamAtMoment: String?
+    pub let primaryPosition: String?
+    pub let height: String?
+    pub let weight: String?
+    pub let totalYearsExperience: String?
+    pub let nbaSeason: String?
+    pub let dateOfMoment: String?
+    pub let playCategory: String?
+    pub let playType: String?
+    pub let homeTeamName: String?
+    pub let awayTeamName: String?
+    pub let homeTeamScore: String?
+    pub let awayTeamScore: String?
+    pub let seriesNumber: UInt32?
+    pub let setName: String?
+    pub let serialNumber: UInt32
+    pub let playID: UInt32
+    pub let setID: UInt32
+    pub let numMomentsInEdition: UInt32?
+
+    init(
+        fullName: String?,
+        firstName: String?,
+        lastName: String?,
+        birthdate: String?,
+        birthplace: String?,
+        jerseyNumber: String?,
+        draftTeam: String?,
+        draftYear: String?,
+        draftSelection: String?,
+        draftRound: String?,
+        teamAtMomentNBAID: String?,
+        teamAtMoment: String?,
+        primaryPosition: String?,
+        height: String?,
+        weight: String?,
+        totalYearsExperience: String?,
+        nbaSeason: String?,
+        dateOfMoment: String?,
+        playCategory: String?,
+        playType: String?,
+        homeTeamName: String?,
+        awayTeamName: String?,
+        homeTeamScore: String?,
+        awayTeamScore: String?,
+        seriesNumber: UInt32?,
+        setName: String?,
+        serialNumber: UInt32,
+        playID: UInt32,
+        setID: UInt32,
+        numMomentsInEdition: UInt32?
+    ) {
+        self.fullName = fullName
+        self.firstName = firstName
+        self.lastName = lastName
+        self.birthdate = birthdate
+        self.birthplace = birthplace
+        self.jerseyNumber = jerseyNumber
+        self.draftTeam = draftTeam
+        self.draftYear = draftYear
+        self.draftSelection = draftSelection
+        self.draftRound = draftRound
+        self.teamAtMomentNBAID = teamAtMomentNBAID
+        self.teamAtMoment = teamAtMoment
+        self.primaryPosition = primaryPosition
+        self.height = height
+        self.weight = weight
+        self.totalYearsExperience = totalYearsExperience
+        self.nbaSeason = nbaSeason
+        self.dateOfMoment= dateOfMoment
+        self.playCategory = playCategory
+        self.playType = playType
+        self.homeTeamName = homeTeamName
+        self.awayTeamName = awayTeamName
+        self.homeTeamScore = homeTeamScore
+        self.awayTeamScore = awayTeamScore
+        self.seriesNumber = seriesNumber
+        self.setName = setName
+        self.serialNumber = serialNumber
+        self.playID = playID
+        self.setID = setID
+        self.numMomentsInEdition = numMomentsInEdition
+    }
+}
+
+pub fun main(address: Address, id: UInt64): TopShotMomentMetadataView {
```

Comment:
return `TopShot.TopShotMomentMetadataView`

Reply:
Done. Thank you.